doc/cli/npm-search.md

```diff
@@ -17,6 +17,20 @@ expression characters must be escaped or quoted in most shells.)
 
 ## CONFIGURATION
 
+### json
+
+* Default: false
+* Type: Boolean
+
+Output search results as a JSON array.
+
+### tab-separated
```

Comment:
I remember we had an over-the-air conversation about `--parseable` vs `--tab-separated` but I don't remember if there was a particular reason for it. So then: would you like me to use `--parseable` instead for this? It's no skin off my back. I have no strong opinion either way, and I understand your point. (and, fwiw, being specific about what the "parseable" format is seems a little nicer, but it's not the convention we already have)